client/src/pages/DisplayEachCountry.js

```diff
@@ -0,0 +1,14 @@
+import DisplayTable from "./DisplayTable.js";
+import "./DisplayData.css";
+
+const DisplayEachCountry = ({ theCountries, theData }) => {
+	const listofCountries = theCountries.map((countryName) => (
+		<div key ={countryName}>
+			<h1 className="header">{countryName}</h1>
+			<DisplayTable countryName={countryName} schoolData={theData}></DisplayTable>
+		</div>
+	));
+	return <div>{listofCountries}</div>;
+};
```

Comment:
Here I think it will be best if we use one table and when it comes to country name we should merge columns `<Td colSpan={5}>` if you have 5 columns. 

Without it, the table looks uneven for each school. 

Another way to fix it would be to set a fixed width for each column, but this will be hard to make responsive (if users vistit from mobile)

Reply:
Used BOX instead however to no little avail. Columns still not aligning properly